UnityProject/Assets/Scripts/Messages/Server/UpdateTileMessage.cs

```diff
@@ -151,93 +112,4 @@ public static void SendTo(GameObject managerSubject, NetworkConnection recipient
 			return msg;
 		}
 	}
-
-	public static class UpdateTileMessageReaderWriters
```

Comment:
UpdateTileMessageReaderWriters?

Reply:
the reader writer massively improves the performance of tile change message,  since it blocks the inclusion of colour and matrix4x4 when appropriate

Reply:
its unused

Reply:
how so? mirror automatically uses it, because it has the 
public static UpdateTileMessage.NetMessage Deserialize(this NetworkReader reader)
and uses it

Reply:
Yeah, mirror uses code generation to use it so will appear unused.